portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase

```diff
@@ -1652,18 +1652,19 @@ definition {
 			PagesAdmin.gotoPageEllipsisMenuItem(
 				menuItem = "Configure",
 				pageName = "Page1");
+		}
 
-			Click(
-				key_text = "Page1",
-				locator1 = "Link#ANY");
+		task ("Then There is warning section right below the Friendly URL field on the form") {
+			AssertTextEquals(
+				locator1 = "Message#ERROR_3",
+				value1 = "Warning:The friendly URL of this page is conflicting with a friendly URL of a page in the site template, from which this site was created: Page Page1 of Site Template Name (Please contact the administrator to resolve this friendly URL conflict.)");
 		}
 	}
 
 	@description = "This is a use case for LPS-174434 TC-8. Verify there's a confirmation dialog when a friendly URL collision exists and Administrator tries to edit an existing Site page."
-	@ignore = "true"
 	@priority = 4
 	test ViewConfirmationDialogOfEditExistingSitePageWithURLCollision {
-		property custom.properties = "feature.flag.LPS-174434=true";
+		property custom.properties = "feature.flag.LPS-174417=true";
```

Comment:
The current test is covered by CPSitetemplates#ViewWarnignMessageOfEditExistingSitePageWithURLCollision. Please remove the current test.

Reply:
ok